target_cratedb/sinks.py

```
@@ -298,10 +300,12 @@ def refresh_table(self, table: Union[sa.Table, str]):
         """
         with self.connector._connect() as connection:
-            if isinstance(table, sa.Table):
+            if isinstance(table, FullyQualifiedName):
+                table_full = str(table)
+            elif isinstance(table, sa.Table):
                 table_full = f'"{table.schema}"."{table.name}"'
             elif isinstance(table, str):
                 table_full = table
             else:
-                raise TypeError(f"Unknown type for `table`: {table}")
+                raise TypeError(f"Unknown type `{type(table)}` for table: {table}")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)